### DISCUSSION TITLE
Including Erosion as a Damage Driver in Coastal Contexts

### DISCUSSION THREAD

**Discussion by HenryGeorgist** [USACE/go-consequences#47] [General]
This thread is designed to open the discussion for how to add Erosion as a damage driver for coastal consequences. We need to discuss the inputs to the process from the perspective of the hazard, the inventory, and then finally the computational sequence for the damage compute.

What are the input hazard parameters (units, representation geospatially, parameter names)?

What are the attributes required for the structure inventory (square footage, building footprint, foundation type)?

How do we complete a compute for damage at a given structure if the above inputs are provided (what is the computational logic)?

**Comment by IdrisDobbs** [comment]
Erosion Hazard Input Parameters - 
This is complex, and is usually defined by a coastal engineer. In general terms coastal erosion is the landward retreat of the shoreline due to numerous coastal processes (waves, tides, etc.) . In Beach-fx we capture this effect as an applied erosion rate, in feet per year which is applied to model reaches, which are typically 500' to 1500' stretches of shoreline.  In addition, there is the erosion caused by the storm event, that alters the beach morphology. Ultimately what is needed is the following:
1) The shoreline erosion relative to the centroid of the damageable asset
2) The vertical depth of the aforementioned erosion.


Attributes for the Structure Inventory
Relevant structure inventory considerations for erosion include:
1)The centroid of the asset located in space,
2) The length and width of the asset
3) The foundation type for the asset
4) The critical erosion threshold depth at which that foundation would begin to be undermined.
5) A damage function describing the relationship between the % of foundation footprint undermined and the loss of structure and content value.


I believe the asset centroid length and width would be used to determine the position of the erosion relative to the asset.  If the erosion meets the foundation critical erosion threshold. then it should be a matter of determining the proportion of the foundation is undermined. The further inland the erosion, the more foundation undermined.

**Reply by HenryGeorgist** [reply]
Can you tell me more about erosion caused by a storm event? Is that additional erosion added on top of an annual loss rate?

**Reply by HenryGeorgist** [reply]
Is there a relationship to the foundation type and the critical erosion depth at which the foundation would begin to be undermined? Or is that asset specific?

**Reply by IdrisDobbs** [reply]
Ok,  So in Beach fx, all assets are interpolated onto cross sectional simplified beach profiles. These profiles consist of an upland, a dune and a berm.  These profiles are subjected to the suite of storms that create a change in the shape of the shoreline. (This is all ran through a shoreline change model to describe all the beach shape change outcomes in response to the storm events. This data is captured in the .sdb input for beach fx and is referenced throughout a model simulation.) So a storm is selected in Beach-fx, the shoreline response is selected via lookup, and then as far as I know, the background erosion rate gets applied.  

In order for a method like this to work with the NSI, it would require transects cut along the ocean facing coastline with each asset linked to transect.

**Reply by HenryGeorgist** [reply]
@IdrisDobbs you are right, there seems to be a connection to a cross sectional geometry describing the dune as part of the data requirement. It seems to be logically connected to the asset, but acted upon by (and impacting the result of) the hazard provider.

**Reply by IdrisDobbs** [reply]
Each foundation type is assigned a critical erosion threshold.  For example, I typically assume a slab foundation would be undermined with 20-24" of erosion, a crawl space or a pier/post would be undermined in 46-48". These foundations derive their "strength" from the bearing capacity of the soil. Once the erosion scours material from underneath the footings, they are compromised. Then there are pile, and helical foundation types which are used in conditions where the bearing capacity of the soil is weak and/or subject to movement. The weight of the earth surrounding the piles is what supports the structure.  SFR on piles are typically assumed to have piles that can be 20 ft or so deep.  We typically assume 10 ft of vertical erosion or more would be needed to undermine the foundations. In the case of high rises built on the ocean front typically have reinforced concrete piles or helicals that can be 2-4' in diameter and up to 50ft deep.  We assume 20-30ft of erosion would be necessary to cause the foundation to fail.

**Comment by HenryGeorgist** [comment]
@IdrisDobbs we are targeting this development with @trietmnj right now. Our current focus is on the percent eroded, percent damage piece of the puzzle and getting that into the logic for the structure damage compute.

We will have to develop a hazard provider to provide the percent eroded. That will be staged as a secondary development in the future, not at the current time.
